nifi-framework-bundle/nifi-framework-extensions/nifi-py4j-framework-bundle/nifi-python-framework/src/main/python/framework/ProcessorInspection.py

```diff
@@ -47,6 +47,70 @@ def visit_Assign(self, node):
                             self.string_assignments.append[variable_name] = string_value
         self.generic_visit(node)
 
+class CollectPropertyDescriptorVisitors(ast.NodeVisitor):
+
+    def __init__(self, module_string_constants):
+        self.module_string_constants = module_string_constants
+        self.discovered_property_descriptors = {}
+        self.logger = logging.getLogger("python.CollectPropertyDescriptorVisitors")
+
+    def resolve_dependencies(self, node: ast.AST):
+        resolved_dependencies = []
+        for dependency in node.elts:
+            variable_name = dependency.args[0].id
+            if not self.discovered_property_descriptors[variable_name]:
+                self.logger.error(f"Not able to find actual property descriptor for {variable_name}, so not able to resolve property dependencies.")
```

Comment:
Any log of this nature should indicate the name of the Processor class that is being parsed. Without it, it's difficult to understand what's happening.